core/services/ocr2/plugins/directrequestocr/aggregation.go

```diff
@@ -0,0 +1,102 @@
+package directrequestocr
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+	"sort"
+
+	"github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr/config"
+)
+
+func CanAggregate(N int, F int, observations []*ProcessedRequest) bool {
+	return len(observations) >= N-F
```

Comment:
If `len(observations)` == 0 and `N` == 0 and `F` == 0, then it will return `true`.
I know this is abnormal case for N, F to be 0 or negative numbers, I hope we verify these somewhere else.

Reply:
Good point, I'll extend this check. N is coming from an array length so it can't be negative. I'm not sure about validating F, though. I remember getting errors about F==0 from a config builder but it might not be verified when received from the contract.